lib/kubernetes-deploy/bindings_parser.rb

```diff
@@ -41,7 +41,7 @@ def parse_file(string)
         when '.json'
           bindings = parse_json(File.read(file_path))
         when '.yaml', '.yml'
-          bindings = YAML.safe_load(File.read(file_path))
+          bindings = YAML.safe_load(File.read(file_path), [], [], true)
```

Comment:
let's pass 'file_path' as the final arg https://ruby-doc.org/stdlib-2.1.0/libdoc/psych/rdoc/Psych.html#method-c-safe_load it should lead to better error messages (the filename gets passed all the way to parse_stream if you don't want to trace what happens).

Reply:
Good idea!